src/components/learner-credit-management/invite-modal/EnterpriseCustomerUserDatatable.jsx

```diff
@@ -0,0 +1,188 @@
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  DataTable,
+  Stack,
+  TextFilter,
+} from '@openedx/paragon';
+import { useEnterpriseLearnersTableData, getAllEnterpriseLearners } from '../data/hooks/useEnterpriseLearnersTableData';
+import { formatTimestamp } from '../../../utils';
+import { DEFAULT_PAGE } from '../data';
+
+const getSelectedEmailsByRow = (selectedFlatRows) => {
+  const emails = [];
+  Object.keys(selectedFlatRows).forEach(key => {
+    const { original } = selectedFlatRows[key];
+    emails.push(original.user.email);
+  });
+  return emails;
+};
+
+const MemberDetailsCell = ({ row }) => (
+  <Stack gap={1}>
+    <div className="font-weight-bold">
+      {row.original.user.username}
+    </div>
+    <div>
+      {row.original.user.email}
+    </div>
+  </Stack>
+);
+
+const MemberJoinedDateCell = ({ row }) => (
+  <div>
+    {formatTimestamp({ timestamp: row.original.created, format: 'MMM DD, YYYY' })}
+  </div>
+);
+
+const AddMembersBulkAction = ({
+  isEntireTableSelected,
+  selectedFlatRows,
+  onHandleAddMembersBulkAction,
+  enterpriseId,
+}) => {
+  const handleOnClick = async () => {
+    let emails = [];
+    if (isEntireTableSelected) {
+      emails = await getAllEnterpriseLearners(enterpriseId);
+      onHandleAddMembersBulkAction(emails);
+      return;
+    }
+    emails = getSelectedEmailsByRow(selectedFlatRows);
+    onHandleAddMembersBulkAction(emails);
+  };
+
+  return (
+    <Button onClick={handleOnClick}>
+      Add
```

Comment:
What I expected was to be able to select members, go to the next page, select more, and then add at the end. It seems to be losing the input from earlier pages when you traverse